cypress/e2e/awx/resources/workflowVisualizerCRUD.cy.ts

```diff
@@ -0,0 +1,459 @@
+import { randomString } from '../../../../framework/utils/random-string';
+import { Inventory } from '../../../../frontend/awx/interfaces/Inventory';
+import { InventorySource } from '../../../../frontend/awx/interfaces/InventorySource';
+import { JobTemplate } from '../../../../frontend/awx/interfaces/JobTemplate';
+import { Organization } from '../../../../frontend/awx/interfaces/Organization';
+import { Project } from '../../../../frontend/awx/interfaces/Project';
+import { WorkflowJobTemplate } from '../../../../frontend/awx/interfaces/WorkflowJobTemplate';
+import { WorkflowNode } from '../../../../frontend/awx/interfaces/WorkflowNode';
+
+describe('Workflow Visualizer', () => {
+  let organization: Organization;
+  let project: Project;
+  let inventory: Inventory;
+  let inventorySource: InventorySource;
+  let jobTemplate: JobTemplate;
+  let workflowJobTemplate: WorkflowJobTemplate;
+  let projectNode: WorkflowNode;
+  let approvalNode: WorkflowNode;
+  let workflowJtNode: WorkflowNode;
+
+  before(function () {
+    organization = this.globalOrganization as Organization;
+    project = this.globalProject as Project;
+    cy.awxLogin();
+
+    cy.createAwxInventory({ organization: organization.id })
+      .then((i) => {
+        inventory = i;
+      })
+      .then(() => {
+        cy.createAwxInventorySource(inventory, project).then((invSrc) => {
+          inventorySource = invSrc;
+        });
+        cy.createAwxJobTemplate({
+          organization: organization.id,
+          project: project.id,
+          inventory: inventory.id,
+        }).then((jt) => (jobTemplate = jt));
+      });
+  });
+
+  beforeEach(function () {
+    cy.createAwxWorkflowJobTemplate({
+      organization: organization.id,
+      inventory: inventory.id,
+    }).then((wfjt) => (workflowJobTemplate = wfjt));
+  });
+
+  afterEach(() => {
+    cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+  });
+
+  after(function () {
+    cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
+    cy.deleteAwxInventorySource(inventorySource, { failOnStatusCode: false });
+    cy.deleteAwxJobTemplate(jobTemplate, { failOnStatusCode: false });
+  });
+
+  describe('Workflow Visualizer: Create', () => {
+    it('should render a workflow visualizer view with multiple nodes present', () => {
+      cy.renderWorkflowVisualizerNodesFromFixtureFile(
+        `${workflowJobTemplate.name}`,
+        'wf_vis_testing_A.json'
+      );
+      cy.get('[class*="66-node-label"]')
+        .should('exist')
+        .should('contain', 'Cleanup Activity Stream');
+      cy.get('[class*="43-node-label"]').should('exist').should('contain', 'bar');
+      cy.get('[class*="42-node-label"]').should('exist').should('contain', '1');
+      cy.get('[class*="41-node-label"]').should('exist').should('contain', 'Demo Project');
+      cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate);
```

Comment:
There are resources being delete as part of the `afterEach`, and `after`. This one seem to be deleted as part of them. Either we create the workflow as part of the test itself, and delete at the end. Or we create as part of a `before`, and delete as part of `after`.  This is a general comment. We might address in a different PR.